packages/react/src/field/Field.tsx

```diff
@@ -103,9 +106,12 @@ export const Field = forwardRef<HTMLDivElement, FieldProps>(
         </FieldContext.Provider>
 
         {error && (
-          <Box asChild color="fg.error" fontSize="sm">
+          <Flex color="fg.error" flexDirection="row" fontSize="sm" gap="4">
+            <Box asChild {...styles.icon()}>
+              <IconTriangleExclamationSmall />
+            </Box>
             {fallbackSpan(error)}
```

Comment:
technically we no longer need to wrap the error message in a span - since we're no longer using box composition here
```suggestion
            {error}
```